src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -79,6 +79,9 @@ message ConfigProto {
 
   // The language of the generated code.
   string language = 5;
+  
+  // The configuration to create the providers.
+  GeneratorProto generator = 6;
```

Comment:
Should we remove the ability to configure `codegen_provider`/`input_element_view`/`snippet_files` altogether?